tests/resources/conformance_suites/nl_nt17/br_kvk/4-17-invalid-period.xbrl

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xbrl
+  xml:lang="en"
+  xmlns="http://www.xbrl.org/2003/instance"
+  xmlns:bzk-wnt-i="http://www.nltaxonomie.nl/nt16/bzk/20211208/dictionary/bzk-wnt-data"
+  xmlns:iso4217="http://www.xbrl.org/2003/iso4217"
+  xmlns:jenv-bw2-dim="http://www.nltaxonomie.nl/nt16/jenv/20211208/dictionary/jenv-bw2-axes"
+  xmlns:jenv-bw2-dm="http://www.nltaxonomie.nl/nt16/jenv/20211208/dictionary/jenv-bw2-domains"
+  xmlns:jenv-bw2-i="http://www.nltaxonomie.nl/nt16/jenv/20211208/dictionary/jenv-bw2-data"
+  xmlns:kvk-i="http://www.nltaxonomie.nl/nt16/kvk/20211208/dictionary/kvk-data"
+  xmlns:link="http://www.xbrl.org/2003/linkbase"
+  xmlns:nl-cd="http://www.nltaxonomie.nl/nt16/sbr/20210301/dictionary/nl-common-data"
+  xmlns:rj-dim="http://www.nltaxonomie.nl/nt16/rj/20211208/dictionary/rj-axes"
+  xmlns:rj-dm="http://www.nltaxonomie.nl/nt16/rj/20211208/dictionary/rj-domains"
+  xmlns:rj-i="http://www.nltaxonomie.nl/nt16/rj/20211208/dictionary/rj-data"
+  xmlns:xbrldi="http://xbrl.org/2006/xbrldi"
+  xmlns:xbrli="http://www.xbrl.org/2003/instance"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+    <link:schemaRef
+      xlink:href="http://www.nltaxonomie.nl/nt16/kvk/20211208/entrypoints/kvk-rpt-jaarverantwoording-2021-nlgaap-klein-publicatiestukken.xsd"
```

Comment:
This is a NL17 conformance suite test case but you're importing NL16 here and in the imports above. You can copy and paste the corresponding lines from most any other existing NL17 test case.